src/App.tsx

```diff
@@ -1,93 +1,101 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Filter } from './types/Filter';
+import { Todo } from './types/Todo';
+import { useLocalStorage } from './UseLocalStorage';
+
+const filteredTodos = (todos: Todo[], filter: string) => {
+  const returnArr = [...todos];
```

Comment:
There is no need to create a copy of the array with 'returnArr' since 'filter' method does not modify the original array. You can directly use 'todos' to filter.